src/test.sh

```diff
@@ -0,0 +1,4 @@
+export PYTHONPATH="${PYTHONPATH}:$HOME/3rdparty/TheanoNetCore/TheanoNetCoresrc"
```

Comment:
It's probably better to put this in a readme file (as this is also specific to your machine).